components/server/ee/src/billing/entitlement-service-ubp.ts

```diff
@@ -97,6 +97,14 @@ export class EntitlementServiceUBP implements EntitlementService {
         return this.hasPaidSubscription(user, date);
     }
 
+    /**
+     * Returns true if network connections should be limited
+     * @param user
+     */
+    async limitNetworkConnections(user: User, date: Date): Promise<boolean> {
+        return !this.hasPaidSubscription(user, date);
```

Comment:
Hehe, this is a TS/JS subtility: This will always be "false".
This works, though:

`        return !(await this.hasPaidSubscription(user, date));`

Reply:
Fixed.